benches/ray_sphere_intersect.rs

```diff
@@ -0,0 +1,241 @@
+#![feature(stmt_expr_attributes)]
```

Comment:
In theory yes, it is there in order to prevent things from getting inlined/autovec'd.

That said, nalgebra wide types require nightly atm so... yeah

Reply:
I'm not sure if we can add attributes only under nightly? i.e. cfg'd attributes... I'll look it up and see

Reply:
Okay I think I figured out a way to make this work with a macro...